ecr/base.go

```diff
@@ -53,25 +52,22 @@ type ecrAPI interface {
 
 func (b *ecrBase) getManifest(ctx context.Context) (*ecr.Image, error) {
 	imageIdentifier := b.ecrSpec.ImageID()
-	log.G(ctx).WithField("imageIdentifier", imageIdentifier).Debug("ecr.base.manifest")
+	log.G(ctx).WithField("imageIdentifier", imageIdentifier).Debug("ecr.base.manifest: getting image")
+
 	batchGetImageInput := &ecr.BatchGetImageInput{
 		RegistryId:     aws.String(b.ecrSpec.Registry()),
 		RepositoryName: aws.String(b.ecrSpec.Repository),
 		ImageIds:       []*ecr.ImageIdentifier{imageIdentifier},
-		// TODO: Determine if this should be hard-coded
```

Comment:
As discussed offline, this PR will drop support for Docker v2 Schema 1 manifests for the time being. Tentatively, support for Schema 1 may return in the form of a functional option to enable its use and conversion to other types. 

In the meantime, the PR is now scoped to the Docker v2 Schema 2 manifest types and the OCI Image Spec types. There's a [static list of the supported manifest types](https://github.com/awslabs/amazon-ecr-containerd-resolver/pull/13/files#diff-c82d190de1354947418a3d1d2ee93a83R36-R42) that lists these and is used to guard against handling other types.

Reply:
One thought I had is that it might be a good idea to have `parseImageManifestMediaType` parse one level deeper into manifest list types to validate that there are *image manifests* listed. However, I'm not sure what's in store with the OCI artifacts spec - it doesn't seem to have any types listed at this time or clarity (to me) on what overlap there is with the OCI image types (or if isn't overlap at all).

Reply:
Note to self: check + update examples and readme to remove the schema 1 conversion.

I'll create a tracking issue to reintroduce Schema 1 support with notes regarding these removed bits once this change is merged.

Reply:
Removed the Schema 1 references in the latest push.

Reply:
> I'm not sure what's in store with the OCI artifacts spec - it doesn't seem to have any types listed at this time or clarity (to me) on what overlap there is with the OCI image types (or if isn't overlap at all).

My understanding is that Artifacts intends to reuse the index and manifest, but not the config or layer media types.  Instead, an artifact author is expected to choose a media type that matches the type of artifact, and is encouraged to register any new media types with IANA.

Reply:
> My understanding is that Artifacts intends to reuse the index and manifest, but not the config or layer media types.

Is there guidance on where the responsibility is on handling these types? Does this fall on the resolver to circuit break? Or is it up to containerd to descend into the manifest and request these opaque "blobs" as needed?

Reply:
Discussed the removal of Schema 1 offline with @samuelkarp and decided that we're going to retain it for the time being. 

The changes to remove (added earlier on in this PR) will be pulled aside to be worked on and fleshed out to permit configuring Schema 1 support. Until then Schema 1 will be enabled inline allowing for local conversion to Schema 2, as it was before.